animareflection/ui
# Toggle Component

## Description

##### Task link: https://trello.com/c/rM43LfM4/124-toggle-component

Added `Toggle` (switch) component to client component core.

## Test Steps

1) Verify structure is sound for primitives, recipe, component, and stories.
2) Verify that tests pass.
3) Verify that `Toggle` renders appropriately in storybook + downstream app(s).
